store/views.py

```diff
@@ -77,6 +104,46 @@ def loanBookView(request):
 @csrf_exempt
 @login_required
 def returnBookView(request):
-    pass
+    response_data = {
+        'message': None,
+    }
+    print(request.POST.get('cid'))
+    copy_id = request.POST.get('cid')
+    c=BookCopy.objects.get(id__exact=copy_id)
+    c.borrow_date=None
+    c.available=True
+    c.borrower=None
+    c.save()
+    response_data['message'] = 'success'
+    return JsonResponse(response_data)
 
+@csrf_exempt
+@login_required
+def rateBookView(request):
+    response_data = {
+        'message': None,
+    }
+    
+    book_id = request.POST.get('bid')
+    rateing = float(request.POST.get('brate'))
+    list_book = Review.objects.filter(book_reviewed=Book.objects.get(id__exact=book_id))
+    list_user_book = Review.objects.filter(Q(book_reviewed=Book.objects.get(id__exact=book_id)) & Q(reviewer=request.user))
```

Comment:
While this is not wrong but `Q` is not needed for such simple queries. You can write even very much complex queries without using `Q`.